docs/pywbemcligeneraloptions.rst

```diff
@@ -593,75 +606,82 @@ CIM_Foo_sub_sub is the subclass of CIM_Foo_sub
 Pywbemcli defined logging
 -------------------------
 
-Pywbemcli provides for logging to either a file or the standard error stream
+Pywbemcli provides logging to either a file or the standard error stream
 of information passing between the pywbemcli client and a WBEM server using the
 standard Python logging facility.
 
 Logging is configured and enabled using the ``--log`` general option on the
 commmand line or the `PYWBEMCLI_LOG` environment variable.
 
 Pywbemcli can log  operation calls that send
-requests to a WBEM server and their responses or the HTTP messages between
+requests to a WBEM server and their responses and the HTTP messages between
 the pywbemcli client and the WBEM server including both the pywbem APIs
 and their responses and the HTTP requests and responses.
 
-The default is no logging if the ``--log`` option is not specified with a
-configuration string.
+The default is no logging if the ``--log`` option is not specified.
 
-The general format of the ``--log`` option is a string with up to 3 fields
-(COMPONENT, DESTINATION, DETAIL):
+The value of the the `--log` option parameter and of the PYWBEMCLI_LOG
+environment variable is a log configuration string with the format defined in
+the ABNF rule LOG_CONFIG_STRING. The log configuration string defines a list of
+one or more log configurations, each with fields COMPONENT, DESTINATION, and
+DETAIL:".
 
 .. code-block:: text
 
-    LOG_CONFIG_STRING := CONFIG[,CONFIG]
-    CONFIG            := COMPONENT"="[DESTINATION[":"DETAIL]
-    COMPONENT         := ('all' / 'api' / 'http')
-    DESTINATION       := ('stderr' / 'file')
-    DETAIL            := ('all'/ 'path'/ 'summary')
+    LOG_CONFIG_STRING := CONFIG[ "," CONFIG]
+    CONFIG            := COMPONENT "=" [DESTINATION[ ":" DETAIL]
+    COMPONENT         := ("all" / "api" / "http")
+    DESTINATION       := ("stderr" / filepath)
```

Comment:
On filepath: Further down it is stated that the log file has a defined name. The example a few ines down uses 'file', so I think `filepath` should be changed to `"file"` (i.e. as a keyword).

Reply:
Will be resolved lateron.